Monika After Story/game/zz_selector.rpy

```diff
@@ -2778,14 +2794,29 @@ label monika_clothes_select:
         )
         sel_map = {}
 
+        def_clothes = [mas_selspr.CLOTH_SEL_MAP["def"]]
+
     # initial dialogue
     m 1hua "Sure!"
 
     # setup the monika expression during the selection screen
     show monika 1eua
 
     # start the selection screen
-    call mas_selector_sidebar_select_clothes(sorted_clothes, mailbox=mailbox, select_map=sel_map)
+    if mas_isMoniLove():
+        # for Love, all unlocked clothes are available
+        call mas_selector_sidebar_select_clothes(sorted_clothes, mailbox=mailbox, select_map=sel_map)
+
+    elif (mas_isMoniEnamored(lower=True) and len(gifted_clothes) == 1) or mas_isMoniNormal(lower=True):
```

Comment:
```suggestion
    elif mas_isMoniNormal(lower=True) or len(gifted_clothes) == 1:
```
since the first `if` checks for Love, by default affection must be enamored or lower. This means  we don't have to check it here with the `len(gifted_clothes) == 1` condition.

Reply:
Actually, that entire elif path is useless now, she changes clothes to def on her own when you get below normal and this topic is aff_ranged to Happy+, so we'll never get here at Normal. Can just elim that and go with the else path